dashboard/test/controllers/api_controller_test.rb

```diff
@@ -1240,7 +1240,7 @@ def get_student_response(script, level, lesson, student_number)
     sign_out :user
     overview_path = 'http://script.overview/path'
     CDO.stubs(:studio_url).returns(overview_path)
-    script = Script.find_by_name('algebra')
+    script = create(:script)
```

Comment:
nit: can you add `with_levels` to this one? I suspect it is testing a bit less functionality when no levels are specified.